src/client/src/rt-theme/ThemeState.tsx

```diff
@@ -7,7 +7,7 @@ import { Theme, themes } from './themes'
 type ThemeName = 'light' | 'dark'
 
 interface ThemeSelector {
-  name?: ThemeName
+  name: ThemeName | null
```

Comment:
As a prop it is not required — it only provides a boundary around which a state may exist.

This would be the meaning of a tear-off being themed differently than the app — the state may only occur once toggled, the effect of that toggle would be limited to that tear-off, at its boundary, the ThemeState.

Reply:
The owner of the ThemeState may of course choose to serialize and provide some state — but it is not required.

Reply:
We can also imagine this a boundary for vendor themes.

Reply:
Leave this optional

Reply:
It is optional. BUt undefined is not a valid value according to the code